k8s-manifests/keda/keda-service-monitor.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: keda-metrics-apiserver-monitor
+  namespace: keda
+  labels:
+    app.kubernetes.io/name: keda-metrics-apiserver
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: keda-metrics-apiserver
+  endpoints:
+  - port: metrics
+    interval: 15s
+
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: keda-operator-monitor
+  namespace: keda
+  labels:
+    app.kubernetes.io/name: keda-operator
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: keda-operator
+  endpoints:
+  - port: metrics
+    interval: 15s
+
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: keda-admission-monitor
+  namespace: keda
+  labels:
+    app.kubernetes.io/name: keda-admission
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: keda-admission
```

Comment:
It seems the `keda-admission-webhooks` svc doesn't have `app.kubernetes.io/name` label, try to use another label?

Reply:
Fixed. Can be checked by prometheus dashboard.